CRM/Api4/Page/AJAX.php

```diff
@@ -81,7 +81,10 @@ public function run() {
       }
     }
     catch (Exception $e) {
-      http_response_code(500);
+      $statusMap = [
+        \Civi\API\Exception\UnauthorizedException::class => 403,
+      ];
+      http_response_code($statusMap[get_class($e) ?? 500]);
```

Comment:
@totten this code looks incorrect - it is defaulting to the array key `500` but the array does not contain that key.
I think it should be
```suggestion
      http_response_code($statusMap[get_class($e)] ?? 500);
```

Reply:
Doh. Thanks @colemanw. Fix: https://github.com/civicrm/civicrm-core/pull/19533